app/views/shared/_title_and_tab_sets.html.erb

```diff
@@ -21,7 +21,9 @@
 
     <!-- Title Caption -->
     <!-- e.g. "Observation 5"; Can be multiple lines, e.g., with Observer ID -->
-    <%= content_tag(:div, @title, id: "title-caption") %>
+    <%= content_tag(:span, @title, id: "title-caption") -%>
+    <%= content_tag_if(@on_obs_filters.present?, :span, "(#{:filtered.t})",
```

Comment:
This should at least have some help text associated with it explaining that it is filtered either by the user's preference or as part of the Advanced Search interface.